### DISCUSSION TITLE
How change colors or customize chatwoot?

### DISCUSSION THREAD

**Discussion by mauromn1992** [chatwoot/chatwoot#6558] [Q&A]
Does anyone have any ideas on how to change the colors and customize the chatwoot style?

Grateful for the attention.

**Comment by 627721565** [comment]
您好，邮件已经收到，查阅后会尽快给您回复！

**Comment by Leandroledledled** [comment]
> Does anyone have any ideas on how to change the colors and customize the chatwoot style?
> 
> Grateful for the attention.

Use https://darkreader.org/, perfect!

**Reply by DAVIAMERICO242** [reply]
are you kidding? it's just for webbrowser

**Comment by aliverio** [comment]
I know how to customize it, by changing some code configuration and compiling the code when modifying vuejs, I could help you but it takes some time, and I'm a bit busy.

**Reply by tobsowo** [reply]
Please, can you share a step by step process or maybe a screen recording.

Thank you.
